packages/rulesets/src/native/utilities/swagger-workspace.ts

```diff
@@ -93,7 +93,7 @@ export namespace Workspace {
     }
     else if(reference){
       const referenceSchema = resolveRef(createEnhancedSchema(model.properties.value.items, source.file), inventory)
-      if (referenceSchema && referenceSchema.value && referenceSchema.value.properties) {
+      if (referenceSchema && referenceSchema.value && (referenceSchema.value.properties || referenceSchema.value.allOf)) {
```

Comment:
Below, the code merges the contents of `model.properties` and `model.allOf`, so this conditional should be true if **either** is defined on `referenceSchema.value`.